sample.env

```diff
@@ -0,0 +1,26 @@
+# Docker compose env file
+# https://docs.docker.com/compose/env-file/
+# https://docs.docker.com/compose/reference/envvars/#compose_file
+COMPOSE_FILE=docker-compose.yml
+COMPOSE_PROJECT_NAME=kaiyuanshe_osschat
+
+#WEBHOOK_PROXY_URL=https://smee.io/pAaHJ4MtZN0K8fb
+#APP_ID=
+#PRIVATE_KEY=
+#WEBHOOK_SECRET=
+
+# Wechaty
+HA_WECHATY_PUPPET=
+HA_WECHATY_PUPPET_TOKEN=
+CHATOPS_ROOM_ID=
```

Comment:
Thanks for adding support to read those values from the environment variables!

Please add the `OSSCHAT_` prefix to those variable names for a better namespaces, and rename them to:

```
OSSCHAT_ROOM_ID_CHATOPS=
OSSCHAT_ROOM_ID_HEARTBEAT=
OSSCHAT_ROOM_ID_DEVELOPER=
```